[style] Admin Page 퍼블리싱

depoo69 opened the pull request:
**📌 관련 이슈번호**
Closes #14 
---

### 체크리스트

- [x]  🎋 base 브랜치를 dev 브랜치로 설정했나요?
- [x]  🖌️ PR 제목은 형식에 맞게 잘 작성했나요?
- [x]  🏗️ 빌드는 성공했나요? (yarn build)
- [x]  🧹 불필요한 코드는 제거했나요? e.g. console.log
- [x]  🏷️ 라벨은 등록했나요?
- [x]  🙇‍♂️ 리뷰어를 지정했나요?

---

## ✅ Key Changes

> 이번 PR에서 작업한 내용을 간략히 설명해주세요
> 

---

**📢 To Reviewers**

---

**📸 스크린샷 or 실행영상**
![image](https://github.com/user-attachments/assets/876ba68e-0dd3-4985-8b40-bdd448ce751d)
![image](https://github.com/user-attachments/assets/db9e14ea-377e-44a7-9d8a-cd27b5f94738)
![image](https://github.com/user-attachments/assets/ed581484-3763-4629-a2af-335fae5b18e1)
![image](https://github.com/user-attachments/assets/6dd70246-36ec-4e82-9bbd-fbc7155ef08f)
![image](https://github.com/user-attachments/assets/f10ddfda-fad1-4be5-b4eb-b64096a4d3f3)
https://github.com/user-attachments/assets/b45082ae-a3b6-4b5c-bd0d-72f267357bd3

depoo69 committed:
[style] Admin Page 퍼블리싱

depoo69 committed:
style : 영화 관리 페이지 퍼블리싱

depoo69 committed:
Merge branch 'dev' of https://github.com/LG-URECA-FINAL-TEAM8/filmeet-frontend into FM-14

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,83 @@
+  font-size: 1rem;
+  border: none;
+  cursor: pointer;
+  font-family: 'SUIT-Bold', sans-serif;
```
font-family  lightTheme 에서 갖다 써야돼요 다른 코드 참고

depoo69 committed:
docs : 주석문 삭제

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,83 @@
+const HeaderButtons = styled.div`
+  display: flex;
+  gap: 1rem; 
+  margin-left: auto;
```
이미 gap 이 1인데 margin - left 필요없을 듯 간격을 주고싶으면 지워보고 똑같으면 코드삭제

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,83 @@
+  margin: 0;
+  font-size: ${(props) => props.fontSize || '1.5rem'};
+  font-weight: bold;
+  font-family: 'SUIT-Bold', sans-serif;
```
여기도 theme 사용해주세요

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,83 @@
+
+export const AdminHeaderText = styled.h1`
+  position: absolute;
+  top: ${(props) => props.top || '50%'}; /* 동적으로 설정 가능한 top */
```
이건 무슨 코드인지 모르겠음 주석 설명을 기능 위주로 적어주면 좋을듯

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,83 @@
+  color: white;
+  font-size: 1.5rem;
+  font-weight: bold;
+  font-family: 'SUIT-Bold', sans-serif;
```
여기도

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,83 @@
+  font-size: 1.5rem;
+  font-weight: bold;
+  font-family: 'SUIT-Bold', sans-serif;
+  box-shadow: 0 0.2rem 0.4rem rgba(0, 0, 0, 0.1);
```
box shadow도 theme 에 정의되어있음 갖다써주세요

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,83 @@
+export const AdminHeader = styled.header`
+  width: 100%;
+  height: 3.875rem;
+  background-color: #2c2c2c;
```
black 컬러도 있어요 theme에

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,83 @@
+
+export const AdminHeader = styled.header`
+  width: 100%;
+  height: 3.875rem;
```
반올림 해서 4rem 해주세요

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,83 @@
+  width: 100%;
+  height: 3.875rem;
+  background-color: #2c2c2c;
+  color: white;
```
theme

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,83 @@
+  text = '관리자 페이지', // 기본 문구
+  fontSize = '1.5rem',
+  textPosition = { top: '50%', left: '20%' }, // 문구 위치 설정
+  showButtons = true,
```
무슨코드? function 안에 스타일 정의하는 건 삼가해주세요

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,83 @@
+
+  return (
+    <AdminHeader>
+      {/* 동적으로 위치 조정 가능한 헤더 문구 */}
```
주석 삭제

gwagjiug commented on src/components/Common/admin/SearchBar.jsx:
```diff
@@ -0,0 +1,51 @@
+  width: 4rem;
+  height: 3.25rem; 
+  background-color: #333;
+  color: white;
```
theme

gwagjiug commented on src/components/Common/admin/SearchBar.jsx:
```diff
@@ -0,0 +1,51 @@
+  background-color: #333;
+  color: white;
+  border: none;
+  border-radius: 0 0.25rem 0.25rem 0;
```
border-radius 는 %로 주세요

gwagjiug commented on src/components/Common/admin/SearchBar.jsx:
```diff
@@ -0,0 +1,51 @@
+  height: 3.25rem; 
+  padding: 0 1rem;
+  font-size: 1rem;
+  border: 0.0625rem solid #ccc; 
```
theme

gwagjiug commented on src/components/Common/admin/List.jsx:
```diff
@@ -0,0 +1,114 @@
+  background-color: transparent;
+  color: #000;
+  border: none;
+  border-radius: 0.25rem;
```
%

gwagjiug commented on src/components/Common/admin/List.jsx:
```diff
@@ -0,0 +1,114 @@
+  background-color: transparent;
+  color: #000;
+  border: none;
+  border-radius: 0.25rem;
```
%

gwagjiug commented on src/components/Common/admin/List.jsx:
```diff
@@ -0,0 +1,114 @@
+`;
+
+const TableRow = styled.tr`
+  border: 0.0625rem solid #ccc;
```
theme

gwagjiug commented on src/components/Common/admin/List.jsx:
```diff
@@ -0,0 +1,114 @@
+  margin-top: 1.5rem;
+  display: flex;
+  justify-content: center;
+  width: 40.75rem; 
```
소수점 제거

gwagjiug reviewed (changes requested):
lightTheme을 하나도 이용안했음 border-radius 같은 애들은 %로 설정하고 theme 에서 사용가능 한 것들은 불러와서 디자인 일관성 지켜주세요~

그리고 rem 소수점도 반올림해서 그냥 제거해주세요

depoo69 committed:
Merge branch 'dev' of https://github.com/LG-URECA-FINAL-TEAM8/filmeet-frontend into FM-14

depoo69 committed:
fix : 전체 코드에 lighttheme 적용

depoo69 committed:
style : lighttheme 적용

depoo69 committed:
style : 소수점 제거

depoo69 committed:
style : 소수점 제거, 테마 적용

gwagjiug reviewed (changes requested):
확인부탁드려요

depoo69 committed:
Merge branch 'dev' of https://github.com/LG-URECA-FINAL-TEAM8/filmeet-frontend into FM-14

depoo69 committed:
style : border에도 lighttheme 적용
feat : 상태 관리 방법 변경, 변수명 변경

depoo69 committed:
style : admin 페이지 컴포넌트화

depoo69 committed:
Merge branch 'dev' of https://github.com/LG-URECA-FINAL-TEAM8/filmeet-frontend into FM-14

depoo69 committed:
delete: AdminMoviePage.jsx, adminmovie.jsx 삭제
feat: AddMovie.jsx, MovieManagement.jsx, LikeManagement.jsx 추가

gwagjiug commented on src/components/Common/admin/AddMovie.jsx:
```diff
@@ -0,0 +1,59 @@
+
+  return (
+    <PageWrapper>
+      <PageTitle>새로운 영화 추가</PageTitle>
```
여기도 src/components/Common/admin/LikeManagement.jsx 처럼 pageTitle로 묶어서 관리하면 좋을 듯?

gwagjiug replied:
아예 한번에 처리해서 prop으로 보내셈

gwagjiug commented on src/components/Common/admin/LikeManagement.jsx:
```diff
@@ -0,0 +1,43 @@
+
+function LikeManagement() {
+
+  const pagetitle = '좋아요 관리';
```
얘를 부모 페이지 전체에서 관리해서 자식 컴포넌트들한테 뿌려주셈

gwagjiug commented on src/components/Common/admin/LikeManagement.jsx:
```diff
@@ -0,0 +1,43 @@
+
+  return (
+    <PageWrapper>
+      <PageTitle>{pagetitle}</PageTitle>
```
받아서 이렇게 처리

gwagjiug commented on src/components/Common/admin/MovieManagement.jsx:
```diff
@@ -0,0 +1,61 @@
+        data={movies}
+        columns={['영화 제목', '수정', '삭제']}
+        actions={[
+          { label: '수정', onClick: (item) => alert(`수정: ${item.title}`) },
```
onClick 이벤트를 함수로 만들어서 함수처리 해주세요

gwagjiug commented on src/components/Common/header/AdminHeader.jsx:
```diff
@@ -0,0 +1,79 @@
+      </HeaderContent>
+      {showButtons && (
+        <ButtonGroup>
+          <StyledButton onClick={onMovieManagementClick}>전체 영화 목록</StyledButton>
```
기본적으로 하드코딩이 너무많음

gwagjiug reviewed (changes requested):
하드코딩이 너무많고, alert 메시지는 온클릭 함수 만들어서 처리하셈

depoo69 committed:
Merge branch 'dev' of https://github.com/LG-URECA-FINAL-TEAM8/filmeet-frontend into FM-14

depoo69 committed:
style: admin page redesign

depoo69 committed:
Merge branch 'dev' of https://github.com/LG-URECA-FINAL-TEAM8/filmeet-frontend into FM-14

depoo69 committed:
style: AdminHeader 수정

depoo69 committed:
Merge branch 'dev' of https://github.com/LG-URECA-FINAL-TEAM8/filmeet-frontend into FM-14

depoo69 committed:
style: Admin page mUI 적용

depoo69 committed:
style: Admin Page 리디자인 및 퍼블리싱

depoo69 committed:
style: 숨김 버튼 디자인 변경

depoo69 committed:
style: AdminHeader 헤더 버튼 반응 추가, 수정/삭제 버튼 간격 조정,
수정 시 title 필드만 수정되도록 변경, 영화 추가 버튼 아이콘 변경,
AdminAddModal에서 장르, 개봉일 선택할 수 있도록 변경

gwagjiug commented on src/assets/svg/Add.jsx:
```diff
@@ -0,0 +1,36 @@
+const SvgAdd = ({ onClick, ...props }) => (
```
svg 변수명 다시 Add가 뭔지 절대 모름 AddButton 이잖음

depoo69 committed:
style: MovieManagement에서 저장 버튼 아이콘으로 변경,
AdminAddModal에서 모달 동작 추가

depoo69 committed:
style: 동작 버튼 MUI 아이콘으로 변경, 헤더 글꼴 변경,
테이블 전체 border 조정, styled-components 객체화,
AdminAddModal ImagePreview 적용

depoo69 committed:
style: AdminAddModal 버튼 디자인 변경

gwagjiug commented on src/components/Common/modal/AdminAddModal.jsx:
```diff
@@ -0,0 +1,208 @@
+  };
+
+  const { isOpen, title, genre, releaseDate } = addModal;
+  const genres = ['액션', '호러', '공포', '코미디',
```
장르 데이터 하나 만들어서 관리부탁

gwagjiug commented on src/components/Common/modal/AdminAddModal.jsx:
```diff
@@ -0,0 +1,208 @@
+        <S.CardContent>
+        <S.ImagePreview>
+          {addModal.image ? (
+            <img
```
밑에서 스타일선언

gwagjiug commented on src/components/Common/modal/AdminAddModal.jsx:
```diff
@@ -0,0 +1,208 @@
+            type="date"
+            label="개봉일 선택"
+            variant="outlined"
+            fullWidth
```
variant , fullwidth 공통되는 건 밑에서 선언해도 될듯

gwagjiug commented on src/components/Common/modal/AdminAddModal.jsx:
```diff
@@ -0,0 +1,208 @@
+          value={title}
+          onChange={(e) => setAddModalTitle(e.target.value)}
+        />
+        <Box sx={{ display: 'flex', gap: '1rem', marginBottom: '1rem' }}>
```
이것도 밑에선언

gwagjiug commented on src/components/Common/admin/MovieManagement.jsx:
```diff
@@ -0,0 +1,229 @@
+  const { currentPage, moviesPerPage, setCurrentPage } = usePaginationStore();
+  const { editingRow, setEditingRow } = useEditStore();
+
+  const tableHeader = {
```
테이블 헤더도 data 폴더에 정의해서 쓸 수 있을듯 다 공통됨

gwagjiug commented on src/components/Common/modal/AdminAddModal.jsx:
```diff
@@ -0,0 +1,208 @@
+    setAddModalReleaseDate,
+  } = useAdminModalStore();
+
+  const modal = {
```
modalContents 등등 네이밍 변경필요

gwagjiug reviewed (changes requested):
수고

depoo69 committed:
style: AdminAddModal 인라인 정의 스타일과
중복되는 스타일 styled-component로 변경,
변수명 조정, genre 부분 데이터 파일을 통한 관리로 변경
영화 관리, 영화 추가, 리뷰 관리에서 중복되는 tableHeader
데이터 파일을 통한 관리로 변경

depoo69 committed:
style: MUI 아이콘 스타일 부분 스타일 컴포넌트화,
AdminAddModal 버튼 스타일 부분 스타일 컴포넌트화

depoo69 committed:
style: pagination 9개에서 7개로 전환
AddMovie CheckBox 기능 추가

depoo69 committed:
Merge branch 'dev' of https://github.com/LG-URECA-FINAL-TEAM8/filmeet-frontend into FM-14

depoo69 committed:
Merge branch 'dev' of https://github.com/LG-URECA-FINAL-TEAM8/filmeet-frontend into FM-14

depoo69 committed:
fix: 영화 추가 부분 검색 후 등록으로 기능 변경

depoo69 committed:
push
